app/views/notifications/denounce_created.html.erb

```diff
@@ -0,0 +1,6 @@
+<h2 style="color:#6610f2">Citizen Denounce</h2>
+
+<h3>
+  A new denounce has been created in your place of interest
+  You can see it <%= link_to("here", denounce_url(@denounce), style: "color:#6610f2") %>
```

Comment:
The style could be called with a class, so as not to place the style in line.

Reply:
Yes, I actually wanted to use bootstrap,  but it seems that is recommended to use inline styles only for email,  https://docs.sendgrid.com/ui/sending-email/cross-platform-html-design.